app.json

```diff
@@ -18,6 +18,14 @@
     "DISQUS_SITE": {
       "required": true,
       "value": "daostack-alchemy-staging"
+    },
+    "BICONOMY_DAPP_ID":{
+      "required": true,
+      "value":"5e961edab93bbb3218f4ca4d"
+    },
+    "BICONOMY_API_KEY":{
+      "required": true,
+      "value":"FnUirn0bX.7e02bb26-8725-45a4-8180-2f67a52871d3"
```

Comment:
Is this meant to be private in any way?  If so then we should just rely on it to be set in Heroku, and not define anywhere in the repo.

Reply:
Yes it's meant to be private